src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs

```diff
@@ -275,14 +283,21 @@ void CheckAssemblyAttributes (AssemblyDefinition assembly, MetadataReader reader
 					case "System.Runtime.Versioning.TargetFrameworkAttribute": {
 							var arguments = attribute.GetCustomAttributeArguments ();
 							foreach (var p in arguments.FixedArguments) {
+								// Of the form "MonoAndroid,Version=v8.1"
 								var value = p.Value?.ToString ();
-								if (value != null && value.StartsWith ("MonoAndroid", StringComparison.Ordinal)) {
```

Comment:
I prefer this original code of using `value.StartsWith("MonoAndroid")` -- though perhaps it should be extended to `value.StartsWith("MonoAndroid,")`? -- largely because I'm not sure about what are the *requirements and constraints* on framework names.

For example, if a "framework name" could contain a comma, the new approach of splitting on all commas may cause a "mismatch".  Using `StartsWith()` against `MonoAndroid,` would alloy such fears.

Reply:
After using `IndexOf` + `Substring`, I think it's still OK to just compare `targetFrameworkIdentifier == "MonoAndroid"`:
```csharp
//
// Summary:
//     Reports the zero-based index of the first occurrence of the specified string
//     in the current System.String object. A parameter specifies the type of search
//     to use for the specified string.
//
// Returns:
//     The index position of the value parameter if that string is found, or -1 if it
//     is not. If value is System.String.Empty, the return value is 0.
public int IndexOf (String value, StringComparison comparisonType);
```
It should be the first instance of `,` and `targetFrameworkIdentifier` will be everything before it.